api/lib/round-tracker.js

```diff
@@ -239,43 +237,38 @@ export async function maybeCreateSparkRound (pgClient, {
   sparkRoundNumber,
   meridianContractAddress,
   meridianRoundIndex,
-  roundStartEpoch,
-  useDynamicTaskCount = true
+  roundStartEpoch
 }) {
   const { rowCount } = await pgClient.query(`
     INSERT INTO spark_rounds
     (id, created_at, meridian_address, meridian_round, start_epoch, max_tasks_per_node)
-    VALUES ($1, now(), $2, $3, $4, $5)
+    VALUES (
+      $1,
+      now(),
+      $2,
+      $3,
+      $4,
+      COALESCE(
+        (SELECT max_tasks_per_node FROM spark_rounds WHERE meridian_round = $3 - 1),
+        $5
+      )
+        * $6
+        / COALESCE(
+            (SELECT measurement_count FROM spark_rounds WHERE meridian_round = $3 - 1),
+            $6
+          )
```

Comment:
I find this bit extremely difficult to follow. Could you please add SQL comments to explain the formula and what $5 & $6 stands for.

Reply:
Is this clearer in https://github.com/filecoin-station/spark-api/pull/385/commits/361f9ee2f7f7d1558f1de1ff4036182b13d12ede?